script.js

```diff
@@ -0,0 +1,432 @@
+$(document).ready(function () {
+
+    var checkIfSpell = true;
+    var classCheck = "";
+    //get all the inputs from the localstorage
+    if (localStorage.length != 0) {
```

Comment:
Always use `!==` so that you compare both the type and the value

There is almost never a good case for using `!=`